bash/lw_aws_inventory.sh

```diff
@@ -12,9 +15,10 @@
 AWS_PROFILE=default
 
 # Usage: ./lw_aws_inventory.sh
-while getopts ":jp:" opt; do
+while getopts ":p:j" opt; do
   case ${opt} in
     p )
+      USE_PROFILE="true"
```

Comment:
Thanks for adding a description of the script options to the PR.
Could we add something to the echo to describe the fields?